edx_sga/tests/test_utils.py

```diff
@@ -35,23 +35,34 @@ def test_utcnow():
     assert is_near_now(now)
     assert now.tzinfo.zone == pytz.utc.zone
 
-@override_settings(SGA_STORAGE_SETTINGS={
-        'STORAGE_CLASS': 'storages.backends.s3boto3.S3Boto3Storage',
-        'STORAGE_KWARGS':
-            {'bucket_name': 'test', 'location': 'abc/def'}
-    })
-def test_get_default_storage_with_settings_override():
+patch("edx_sga.utils.configuration_helpers")
+def test_get_default_storage_with_settings_override(mock_configuration_helpers):
     """
     get_default_storage should return an S3Boto3Storage object
     """
+    mock_configuration_helpers.return_value.get_value.return_value = (
+        {
+            "SGA_STORAGE_SETTINGS": {
+                "STORAGE_CLASS": "storages.backends.s3boto3.S3Boto3Storage",
+                "STORAGE_KWARGS": {
+                    "access_key": "test_key",
+                    "secret_key": "test_secret",
+                    "bucket_name": "test-bucket-1",
+                    "region_name": "us-east-1",
+                }
```

Comment:
Trailing coma. @Nekenhei

Reply:
We have defined it as good practice to apply trailing commas regardless of the case following PEP8.
Trailing commas have nothing to do with the fact that you don't expect the list to grow, trailing commas are a convention for version control systems, not for unit tests, so applying trailing commas does not depend on the test or any other implementation.
We have different practices than upstream, so we should follow our own practices and not depend on others.